Client/mods/deathmatch/logic/CClientRadarAreaManager.h

```diff
@@ -36,6 +36,9 @@ class CClientRadarAreaManager
     unsigned short GetDimension(void) { return m_usDimension; };
     void           SetDimension(unsigned short usDimension);
 
+    unsigned char GetInterior(void) { return m_ucInterior; };
```

Comment:
Remove the **void** inside the parentheses and make the method const: `... GetInterior() const ...`